api/src/main/java/org/openmrs/module/kenyaemr/reporting/library/specialClinics/SpecialClinicsCohortLibrary.java

```diff
@@ -90,20 +81,38 @@ public CohortDefinition totalNumberOfATsDispensed(int intervention, int referred
         return cd;
     }
 
-public CohortDefinition neurodevelopmental(String specialClinic) {
-    SqlCohortDefinition cd = new SqlCohortDefinition();
-    String sqlQuery = "select he.patient_id from kenyaemr_etl.etl_special_clinics he where he.neuron_developmental_findings in (152492,144481,117470,126208) and he.special_clinic = '" + specialClinic + "' and he.visit_date between date(:startDate) and date(:endDate);";
-    cd.setName("Neurodevelopmental");
-    cd.setQuery(sqlQuery);
-    cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
-    cd.addParameter(new Parameter("endDate", "End Date", Date.class));
-    cd.setDescription("Neurodevelopmental");
-    return cd;
-}
+    public CohortDefinition neurodevelopmental(String specialClinic) {
+        SqlCohortDefinition cd = new SqlCohortDefinition();
+        String sqlQuery = "SELECT DISTINCT v.patient_id\n" +
+                "FROM kenyaemr_etl.etl_special_clinics v\n" +
+                "WHERE v.neuron_developmental_findings LIKE '%Cerebral palsy%'\n" +
+                "   OR v.neuron_developmental_findings LIKE '%Down syndrome%'\n" +
+                "   OR v.neuron_developmental_findings LIKE '%Hydrocephalus%'\n" +
+                "   OR v.neuron_developmental_findings LIKE '%Spina bifida%'\n" +
+                "   AND v.visit_date between date(:startDate) and date(:endDate)\n" +
+                "   AND v.special_clinic_form_uuid = '" + specialClinic + "';";
+        cd.setName("Neurodevelopmental");
+        cd.setQuery(sqlQuery);
+        cd.addParameter(new Parameter("startDate", "Start Date", Date.class));
+        cd.addParameter(new Parameter("endDate", "End Date", Date.class));
+        cd.setDescription("Neurodevelopmental");
+        return cd;
+    }
 
     public CohortDefinition learningFindings(String specialClinic) {
         SqlCohortDefinition cd = new SqlCohortDefinition();
-        String sqlQuery ="select he.patient_id from kenyaemr_etl.etl_special_clinics he where he.visit_date between date(:startDate) and date(:endDate) and he.learning_findings in (118795, 118800, 141644,153271,121529,155205,126456) and he.special_clinic = '" + specialClinic + "';";
+        String sqlQuery = "SELECT DISTINCT v.patient_id\n" +
+                "FROM kenyaemr_etl.etl_special_clinics v\n" +
+                "WHERE v.learning_findings LIKE '%Dysgraphia%'\n" +
```

Comment:
Use in instead of like (%%). For example : v.learning_findings in ('Nonverbal learning disabilities','Dyscalculia','Dyslexia') since you are getting data from the same column.